package.json

```diff
@@ -20,8 +20,8 @@
     "@babel/runtime": "7.9.2",
     "@fortawesome/fontawesome-free": "5.13.0",
     "acorn": "7.1.1",
-    "adhocracy4": "liqd/adhocracy4#aplus-v2002.4",
-    "autoprefixer": "9.7.5",
+    "adhocracy4": "liqd/adhocracy4#5f83e0a2970c79476206d4d0830e1a3cc265e73a",
+    "autoprefixer": "9.7.4",
```

Comment:
Is this on purpose or ended up in here by mistake?

Reply:
the auto prefixer no, but the hash I left as need to update it anyway whic I will do now